path_role.go

```diff
@@ -120,7 +120,7 @@ Defaults to 60 (1 minute) if set to 0 and can be disabled if set to -1.`,
 				Type:        framework.TypeCommaStringSlice,
 				Description: `Comma-separated list of 'aud' claims that are valid for login; any match is sufficient`,
 			},
-			"bound_audience_disregard_trailing_slash": {
+			"normalize_bound_audiences": {
```

Comment:
Should we return this flag in pathRoleRead()? Otherwise how do we check if it's set?